package.json

```diff
@@ -17,7 +17,9 @@
     "test": "react-scripts test",
     "eject": "react-scripts eject",
     "jest": "jest --watch a",
-    "cypress": "npx cypress run"
+    "cypress-component": "npx cypress open --component --browser chrome",
```

Comment:
Good work! 
One minor note: replacing `open` with `run` in this command enables us to run tests in the console without CLI.